src/os/bluestore/BlueStore.cc

```diff
@@ -9691,7 +9691,7 @@ int BlueStore::_generate_read_result_bl(
     const BlobRef& bptr = b2r_it->first;
     regions2read_t& r2r = b2r_it->second;
     dout(20) << __func__ << "  blob " << *bptr << std::hex
-             << " need 0x" << r2r << std::dec << dendl;
+             << " need " << r2r << std::dec << dendl;
```

Comment:
could you please drop the `std::hex` and `std::dec` above?

Reply:
yeah, I'm doing it.